test_data/conus_2yr/conus_2yr.yaml

```diff
@@ -23,19 +29,30 @@ test_ans:
     end_time: 1980-12-31T00:00:00
     initial_deltat: 24
 
+  parameter_read:
+    nhru: 765
+    ngw: 765
+    nssr: 765
+
   preprocess_cbh:
     files_to_df:
-      prcp: 0.09595891072952142
-    df_concat: 41.3837384054782
+      prcp: 0.12495362248866715
```

Comment:
I think at some point we need a better way of evaluating a comparison than baking the results into the yaml. It was a relatively involved process to do this when I added the UCB model.